### ISSUE TITLE
Adding a map 

### ISSUE THREAD

**Issue by tapasweni-pathak** [systers/vms#77]
@willingc How about showing a small map to the user, in features like create event, where we have something as 'location'? 

@vubo  What do you think about it? There are packages that we can easily use to implement this feature.

**Comment by vubo** [comment]
@tapasweni-pathak great, I was also planning to do this in the future. What packages can you advise?

**Comment by tapasweni-pathak** [comment]
I have used [Django Easy Maps](https://pypi.python.org/pypi/django-easy-maps) in past. It's good. I think it fits are current needs. Just check it, if it is fine.

**Comment by smarshy** [comment]
@tapasweni-pathak @vubo Do you still need this feature? I have worked with Easy Maps in the past. I could try implementing this.

**Comment by vubo** [comment]
@smarshy sounds great :star: What do you think @tapasweni-pathak ?

**Comment by tapasweni-pathak** [comment]
Sure @smarshy, please go ahead.

**Comment by smarshy** [comment]
Great! I will try to put this up soon :smiley:

**Comment by smarshy** [comment]
@tapasweni-pathak @vubo  Do you need this for jobs too? I could not find a location field for it.
I have done this for events and shifts.

![www gifcreator me_iqhvo1](https://cloud.githubusercontent.com/assets/13931206/12866818/cffdc124-cd01-11e5-9919-7b3596f1c3cb.gif)

Please let me know what you think.

**Comment by tapasweni-pathak** [comment]
@smarshy If there is no location related field, then there is no need to add this. It looks great btw. But this is one feature which needs a lot of testing. :smile:

**Comment by smarshy** [comment]
@tapasweni-pathak I agree. I ran into plenty of errors during the process itself :worried: 
I have tried to resolve them and made the pull request. Please have a look whenever free. Thanks!
